functions/auth.js

```diff
@@ -23,7 +23,7 @@ const {JWT} = require('google-auth-library');
 
 const API_SCOPES = ['https://www.googleapis.com/auth/spreadsheets'];
 
-class GndAuth {
+class Auth {
```

Comment:
If we are removing 'gnd' prefix here, are we removing 'gnd' prefix for other variables like GndFocusableRow & GndFormEditor ect. ?

Reply:
See comment below - basically this is a normal class, while the Gnd* classes in web are React components, which might be treated as a special case. I think it's probably unnecessary for those as well, I think it might be a pattern I carried over from the Polymer 2 impl.